data.sql

```diff
@@ -17,7 +17,7 @@ VALUES
 INSERT INTO timesheet(project_id,employee_id,name,description,allocation,created,updated)
 VALUES
     (1,1,'test','testing',100,CURRENT_TIMESTAMP, CURRENT_TIMESTAMP ),
-    (1,1,'test2','testing delete',100,CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+    (1,2,'test2','testing delete',100,CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
```

Comment:
Add the status field to the timesheet inserts too?

Reply:
Yes.